acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/clique/CliqueMiningAcceptanceTest.java

```diff
@@ -72,6 +72,8 @@ public void shouldStallMiningWhenInsufficientValidators() throws IOException {
     cluster.stopNode(minerNode2);
     cluster.stopNode(minerNode3);
     minerNode1.verify(net.awaitPeerCount(0));
+    minerNode1.verify(clique.blockIsCreatedByProposer(minerNode1));
```

Comment:
does this still work if minerNode1 has already proposed a block before 2 & 3 are stopped?

Reply:
I think it does. In the case where miner 1 has already proposed a block before 2 & 3 are stopped and the head hasn't moved then it then will just pass the verify immediately and then wait for no more blocks to be proposed by 1. If the head has moved as 2 & 3 are being stopped then the verify will wait for miner 1 to mine another block which it is allowed to do as clique only cares about the block proposer not peers.